demisto_sdk/commands/common/hook_validations/pack_unique_files.py

```diff
@@ -308,11 +307,10 @@ def validate_version_bump(self):
         current_version = current_meta_file_content.get('currentVersion', '0.0.0')
         if LooseVersion(old_version) < LooseVersion(current_version):
             return True
-
-        elif self._add_error(Errors.pack_metadata_version_should_be_raised(self.pack, old_version), metadata_file_path):
+        elif self._add_error(Errors.pack_metadata_version_should_be_raised(self.pack, old_version),
+                             metadata_file_path):
             return False
-
-        return True
+        return False
```

Comment:
Why was this changed?

Reply:
you are right i did there some modification that was necessary in the end and this remain i will change it back. 
although it is i strange code and will be happy to go over with you.